docs/api-testing/variables-environment-overrides.md

```diff
@@ -60,24 +62,28 @@ Yes, we're using variables as expected values.
 
 Explore our variable containers from the lowest to the highest priority:
 
+### Company Vault
+The [**Company Vault**](/api-testing/vault/#company-vault) is where you can store variables and snippets to use across globally (i.e., across all of your Projects).
+
 ### Project Vault
-Each Project has a **Vault** where you can save Project-specific variables and snippets. It is global to the Project (i.e., across all tests) and its variables are injected during test execution.
+The [Project **Vault**](/api-testing/vault/#project-vault) is where you can save variables and snippets that will be specific to that Project (i.e., across all tests). Its variables are injected during test execution.
 
 ### Globals/Input Set
-These are _test__-specific and contain the variables needed by the test to perform its duties.
+These are _test_-specific and contain the variables needed by the test to perform its duties.
 
 Since Globals/Input Set variables can be overridden to hit different Environments, we recommend considering these variables as containers of the default scenario.
 
 If the same variable exists in both in the **Globals/Input Set** and Project-specific **Vault**, the one used in the **Globals/Input Set** will win over the **Vault**.
 
-### Overriding Variables
+### Overriding Variables, Environments
 When you declare an overridden variable (using the API, [apifctl command line utility](/api-testing/apifctl-cicd-integration/), or the [**Schedule** tool](/api-testing/schedule-a-test/)) its value will be injected into the test when it's executed. If the variable has already been declared in the **Vault** or the **Globals/Input Set**, it will be rewritten with the new value.
 
-### Environments and Presets
-They are collections of overrides. You can save an environment with a name and reuse it when running a test.
+Environments are collections of overrides. You can save an environment as a preset with a name and reuse it when running a test.
+
+### `Set` Variable Component
+The last item of the chain, the `Set` commands within a test, will introduce a runtime variable. If a variable with that name has already been declared, it will be overwritten by this.
 
-### `SET` commands
-The last item of the chain, the `SET` commands within a test will introduce a runtime variable. If a variable with that name has already been declared, it will be overwritten by this.
+From a test, go to **Compose** > click **Add Component** (**+** icon) > click the **Set (variable)** component > enter your **Var**, **Variable Mode** (data, string, or langauge) and **Value** > click the checkmark icon to add to your test.
```

Comment:
@LoryApiFortress When you get a sec, could you pls review the changes I've made to the Set (variable) section?

Reply:
@kimsauce you misspelled "language" (...**Variable Mode** (data, string, or langauge)...)
To answer your question:
it's not completely true. "var" field and "variable mode" are always shown, the "value" field is available only if you choose "string" as "variable mode". If you choose "data" you will have "data" as text area and if you choose "language" it will presents you "lang" drop down (available options are "javascript" and "template") and "content" text area, but you can keep it simple adding something like "fill out all the available fields"

Reply:
Hi @LoryApiFortress ! I had fixed the spelling shortly after -- it's in prod https://docs.saucelabs.com/api-testing/variables-environment-overrides/#set-variable-component. That's weird that it didn't update here! I used the same branch.

I'll fix this --> it's not completely true. "var" field and "variable mode" are always shown, the "value" field is available only if you choose "string" as "variable mode". If you choose "data" you will have "data" as text area and if you choose "language" it will presents you "lang" drop down (available options are "javascript" and "template") and "content" text area, but you can keep it simple adding something like "fill out all the available fields"